api-tester/src/main/java/com/revenuecat/apitester/kotlin/PurchasesAPI.kt

```diff
@@ -25,6 +24,7 @@ import com.revenuecat.purchases.interfaces.ReceiveOfferingsCallback
 import com.revenuecat.purchases.interfaces.UpdatedCustomerInfoListener
 import com.revenuecat.purchases.logInWith
 import com.revenuecat.purchases.logOutWith
+import com.revenuecat.purchases.models.BillingFeature
 import com.revenuecat.purchases.models.PurchaseOption
 import com.revenuecat.purchases.models.StoreProduct
 import com.revenuecat.purchases.models.StoreTransaction
```

Comment:
This file should have the contents of the removed PurchasesBC5API.kt

Reply:
i made a BillingFeatureAPI file instead, was gonna split out the BC5 ones into files eventually

Reply:
I see. I like it! Thanks